federator/src/main.js

```diff
@@ -49,6 +60,16 @@ async function run() {
     try {
         await mainFederator.run();
         await sideFederator.run();
+
+        if (config.mainchain.nftBridge !== null) {

```

Comment:
add a check that if config.mainchain.nftBridge and config.sideChain.nftBridge is not defined throw an error and viceversa

Reply:
done at https://github.com/rsksmart/tokenbridge/pull/257/commits/960a77254356ad684efe45071b291b518e0266f8